### PR TITLE
Issue:256 Add documentation for external dns

### LINKED ISSUE #256: Update documentation for External Access to include External DNS
### Description
External Access documentation does not have details about how external dns can be configured and used by clients.
This needs to be added.

### Importance
must-have

### PR DESCRIPTION
**Pull request by pbelgundi**
Signed-off-by: pbelgundi <prajakta.belgundi@emc.com>

### Change log description
Add documentation for segment store external dns to operator external access documentation.

### Purpose of the change
Fixes #256

### CODE REVIEW

File: doc/external-access.md [hunk 1]
```diff
@@ -115,3 +115,41 @@ example-pravega-segmentstore-headless   ClusterIP      None            <none>
 ```
 
 In the example above, clients will connect to the Pravega Controller at `tcp://35.239.48.145:9090`.
+
+# Using External DNS names for segment store pods
+
+Given a domain suffix for the cluster, each segment store pod can be assigned a DNS name. An external DNS service can resolve this DNS name to the pod externalIP.
+Domain name can be provided in the manifest using `domainname` key under `externalAccess`.
+Segment store dns name is created by prefixing the domain name with segmentstore pod-name.
```
Comment (Tristan1900) [inline doc/external-access.md]: Some minor issues. The use of word `segmentstore` is not consistent in this doc, some with space in between and some are not. I think we probably should use `segmentstore` without a space.

Comment (pbelgundi) [reply doc/external-access.md]: Sure will fix these..

Comment (pbelgundi) [reply doc/external-access.md]: Done.

**Comment by Tristan1900** [comment]
LGTM

**Pull request merged**